.github/workflows/build_deploy_wheels.yml

```diff
@@ -43,3 +43,41 @@ jobs:
           TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
         run: |
           twine upload dist/*-manylinux*.whl
+
+  deploy_wheels_windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    name: Building for windows ${{ matrix.python-version }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+
+      - name: Install system dependencies
+        run: choco install ngspice
```

Comment:
Probably you don't need `ngspice` to build the wheels for Windows.

Reply:
Done :rocket: